ci-scripts/release-tests.sh

```diff
@@ -36,8 +41,27 @@ export RHDH_HELM_REPO='$RHDH_HELM_REPO'
 
     git commit -am "chore($ticket): $testname on $branch"
     git push -u origin "$branch"
-    echo "$(date -Ins --utc) INFO Create PR on https://github.com/redhat-performance/backstage-performance/pull/new/${branch} and comment '/test mpc' there"
-    git checkout main
+    echo "$(date -Ins --utc) INFO Created and pushed branch ${branch}"
+    git checkout "$SOURCE_BRANCH"
+
+    curl_data='{
+        "title": "chore('"$ticket"'): '"$branch"'",
+        "body": "**'"$testname"'**: '"$VERSION_OLD"' vs. '"$VERSION_NEW"' testing. This is to get perf&scale data for `'"$branch"'`",
+        "head": "'"$branch"'",
+        "base": "'"$SOURCE_BRANCH"'",
+        "draft": true
+    }'
+    curl_out="$( curl \
+        -L \
+        --silent \
+        -X POST \
+        -H "Accept: application/vnd.github+json" \
+        -H "Authorization: Bearer $GITHUB_TOKEN" \
+        -H "X-GitHub-Api-Version: 2022-11-28" \
+        "https://api.github.com/repos/redhat-performance/backstage-performance/pulls" \
+        -d "$curl_data"
+    )"
+    echo "$(date -Ins --utc) INFO Created PR $( echo "$curl_out" | jq .html_url ) so you can comment '/test mvp-scalability' there"
```

Comment:
@jhutar Consider adding the comment as part of the script
```suggestion
    echo "$(date -Ins --utc) INFO Created PR $( echo "$curl_out" | jq .html_url ) so you can comment '/test mvp-scalability' there"
```
```suggestion
    pr_number=$( echo "$curl_out" | jq -rc '.number' )
    curl_comment_out="$( curl \
        -L \
        --silent \
        -X POST \
        -H "Accept: application/vnd.github+json" \
        -H "Authorization: Bearer $GITHUB_TOKEN" \
        -H "X-GitHub-Api-Version: 2022-11-28" \
        "https://api.github.com/repos/redhat-performance/backstage-performance/issues/$pr_number/comments "
        -d '{"body":"/test mvp-scalability"}'
```